File: Praktikum Python/PRAK204-2210817210033-HariOctavianDelrossi.py
pi=3.142857

r=int(input('Masukkan jari-jari: '))
t=int(input('Masukkan tinggi: '))

v=pi*r*r*t
print('\nVolume= ',end='')
print('%.2f' %v)
l=2*pi*r*(r+t)
print('Luas= ',end='')
print('%.2f' %l)
k=2*pi*r
print('Keliling= ',end='')
print('%.2f' %k)

File: Praktikum C/PRAK204-2210817210033-HariOctavianDelrossi.c
#include <stdio.h>
int main(){
    float r,t,v,l,k;
    float pi=3.142857;
    printf("Masukkan jari-jari: ");scanf("%f",&r);
    printf("Masukkan tinggi: ");scanf("%f",&t);
    printf("\n");
    v=pi*r*r*t;
    printf("Volume= %.2f\n",v);
    l=2*pi*r*(r+t);
    printf("Luas= %.2f\n",l);
    k=2*pi*r;
    printf("Keliling= %.2f\n",k);
    return 0;
}